src/Impostor.Server/Net/State/Game.Data.cs

```diff
@@ -315,7 +315,18 @@ public async ValueTask<bool> HandleGameDataAsync(IMessageReader parent, ClientPl
 
                     default:
                     {
-                        _logger.LogWarning("Bad GameData tag {0}", reader.Tag);
+                        if (Enum.IsDefined(typeof(GameDataTag), reader.Tag))
```

Comment:
GameDataTag is not an enum, does this work?

Reply:
This fix is dumb, but i dont know better ways to do this